### ISSUE TITLE
"Enter the Public URL of the Posit Connect Server" is a little too wordy

### ISSUE THREAD

**Issue by jonkeane** [posit-dev/publisher#2371]
One location: https://github.com/posit-dev/publisher/blob/7f8677442ea18c17863808d1ffcd45abc53da847/extensions/vscode/src/multiStepInputs/newCredential.ts#L87-L88

Consider changing to "Enter the URL of the Posit Connect server" not sure that 'public' is helping here. I also wonder if this whole sequence should have "Enter the..." (do other steppers say the same thing? or do they just start with the noun phrase?)

**Comment by dotNomad** [comment]
For reference:

Some of the included input prompts in VS Code are a bit wordy with a "Please provide..." at the beginning

![Image](https://github.com/user-attachments/assets/c5dbac5c-f855-4d60-9ada-718eee34fe25)

It looks like leading with the verb (after a "please") is the go to for VS Code.

GitLens uses the place holder and starts with "Enter...":

![Image](https://github.com/user-attachments/assets/ed66f9e2-54ed-45bd-b0f6-7be633b2a934)

**Comment by sagerb** [comment]
I went with:
```
prompt: "Please provide the Posit Connect server's URL",
placeholder: "Server URL",
```
